upgrade/1.4.4/README.md

```diff
@@ -312,10 +312,9 @@ In lieu of rebuilding the storage nodes, they will be live patched.
 
    ```bash
    if grep -q '^export CSM_ARTI_DIR=' /etc/cray/upgrade/csm/myenv ; then
```

Comment:
This `if` statement may not be needed at all

Reply:
True. Fixed.